backend/examples/summa_solvency_flow.rs

```diff
@@ -47,30 +49,24 @@ async fn main() -> Result<(), Box<dyn Error>> {
 
     println!("1. Ownership proofs are submitted successfully!");
 
-    // 2. Submit solvency proof
+    // 2. Submit Commitment
     //
-    // Initialize the `Round` instance to submit the proof of solvency.
-    let asset_csv = "src/apis/csv/assets.csv";
-    let entry_csv = "../zk_prover/src/merkle_sum_tree/csv/entry_16.csv";
+    // Initialize the `Round` instance to submit the liability commitment.
     let params_path = "ptau/hermez-raw-11";
+    let assets_csv_path = "src/apis/csv/assets.csv";
 
-    // Using the `round` instance, the solvency proof is dispatched to the Summa contract with the `dispatch_solvency_proof` method.
-    let mut round = Round::<4, 2, 14>::new(
-        "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80", // anvil account [0]
-        anvil.chain_id(),
-        anvil.endpoint().as_str(),
-        AddressInput::Address(summa_contract.address()),
-        entry_csv,
-        asset_csv,
-        params_path,
-        1,
-    )
-    .unwrap();
+    let entries = get_sample_entries();
```

Comment:
Can we add `entry.csv` file back here? I think there was a misunderstanding in one of my previous comments. I didn't mean to remove the usage of `entry.csv`. I just meants that this should no longer be an input for the `Round`  builder. But it's still ok to me to build the `mst` using `let merkle_tree =MerkleSumTree::new("src/merkle_sum_tree/csv/entry_16.csv")` and then pass it to `Round`

Reply:
Updated at [5a9120f](https://github.com/summa-dev/summa-solvency/pull/179/commits/5a9120f25dfebf26826ba5799e4a53c571e8cf06)